Parser/pgen/keywordgen.py

```diff
@@ -0,0 +1,61 @@
+"""Generate Lib/keyword.py from the Grammar and Tokens files using pgen"""
+
+import argparse
+
+from .pgen import ParserGenerator
+
+TEMPLATE = r'''\
+#! /usr/bin/env python3
+
+"""Keywords (from "Grammar/Grammar")
+
+This file is automatically generated; please don't muck it up!
+
+To update the symbols in this file, 'cd' to the top directory of
+the python source tree and run:
+
+    python -m Parser.pgen.keywordgen ./Grammar/Grammar \
+                                     ./Grammar/Tokens \
+                                     ./Lib/keyword.py
+
+Alternatively, you can run 'make regen-keyword'.
+"""
+
+__all__ = ["iskeyword", "kwlist"]
+
+kwlist = [
+    {keywords}]
+
+iskeyword = frozenset(kwlist).__contains__
+'''
```

Comment:
hack: you can call .lstrip() here :-)

Reply:
I find that a bit ugly, but I have done it in 959cee2 ;)